src/collective/cover/tests/test_basic_tile.robot

```diff
@@ -186,6 +197,16 @@ Test Basic Tile
     # check for successful AJAX refresh
     Wait Until Page Contains  ${title_sample}
 
+    Go to  ${edit_url}
```

Comment:
add a comment to document why you are doing this and what are you trying to demonstrate.